shielddash/core/static/js/app/components/views/fetch-error.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export default function(props) {
   return (
     <dl className="component-error-message">
-      <dt>{props.status}</dt>
+      <dt>{props.status} - {props.detail}</dt>
```

Comment:
The detail would usually go into the `<dd>`. There is nothing programmatically wrong with this approach though I am not sure it addresses lonnen's concern entirely. We'll also need to check if the email domain is anything other than `@mozilla.com` and show a message that reflects that.